Source/Dafny/Verifier/Translator.ExpressionTranslator.cs

```diff
@@ -110,6 +111,10 @@ public ExpressionTranslator(ExpressionTranslator etran, string modifiesFrame)
         Contract.Requires(modifiesFrame != null);
       }
 
+      internal IToken GetTok(Expression e) {
```

Comment:
Why not name it `GetToken` ? Also, I'm not a fan of `e` even though this is such as short method ^^